docs/theme.config.js

```diff
@@ -38,6 +39,23 @@ const theme = {
       titleTemplate: `%s – ${section}`,
     };
   },
+  gitTimestamp({ timestamp }) {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
I can't fix this lint problem; it's complaining that `gitTimestamp` isn't `GitTimestamp`, but that's a Nextra configuration thing.